Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertPipe.cs

```diff
@@ -114,12 +127,13 @@ public ApplicationObject PipeToNative(BuiltElements.Pipe specklePipe)
       }
 
       if (speckleRevitPipe != null)
+      {
         SetInstanceParameters(pipe, speckleRevitPipe);
+        CreateSystemConnections(speckleRevitPipe.Connectors, pipe, receivedObjectsCache);
```

Comment:
This is the actual bug fix: for generic mep elements, `speckleRevitPipe` is null